Classes/common/HTTP/CDTURLSessionTask.m

```diff
@@ -18,6 +18,12 @@
 #import "CDTHTTPInterceptor.h"
 #import "CDTLogging.h"
 
+@interface CDTHTTPInterceptorContext ()
+
+@property (nullable, nonatomic, strong) NSURLResponse *response;
```

Comment:
I don't think we should allow them to do that, I'm pretty sure we don't provide that ability in Java to change the response received, the response should be the response, and the interceptors can do stuff based on it.